arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "version": "2017-06-01",
```

Comment:
Yes, 2017-06-01 is our GA version, and after GA we won't be having any major changes(atleast all major changes are already being covered) and if something new comes after GA, it will be part of new release(not planned as of now), we will be bumping up the version in that case. I hope, that will be fine.